packages/owi/owi.0.2/opam

```diff
@@ -0,0 +1,82 @@
+opam-version: "2.0"
+synopsis:
+  "OCaml toolchain to work with WebAssembly, including an interpreter"
+description:
+  "owi is an OCaml toolchain to work with WebAssembly. It provides an interpreter as an executable and a library."
+maintainer: ["Léo Andrès <contact@ndrs.fr>"]
+authors: [
+  "Léo Andrès <contact@ndrs.fr>"
+  "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+  "Filipe Marques <filipe.s.marques@tecnico.ulisboa.pt>"
+  "Eric Patrizio <epatrizio@mpns.fr>"
+  "Arthur Carcano <arthur.carcano@ocamlpro.com"
+]
+license: "AGPL-3.0-or-later"
+tags: ["owi" "ocaml" "webassembly" "wasm" "interpreter" "compiler"]
+homepage: "https://github.com/ocamlpro/owi"
+bug-reports: "https://github.com/ocamlpro/owi/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml" {>= "5.1"}
+  "integers" {>= "0.5.1"}
+  "cmdliner"
+  "conf-clang"
+  "conf-python-3-dev"
+  "sedlex"
+  "menhir" {build & >= "20220210"}
```

Comment:
Are you sure you want this to be a `build` dependency? The generated code from Menhir can diverge and cause the build to be inconsistent on an upgrade.

Reply:
Yes, it already was a build dependency on previous release and we never had an issue so it should be just fine.

Reply:
I took a quick look at why this works currently, and it's only because `menhir` has a strong dependency on `menhirLib` which is itself not a build dependency.